R/auto.R

```diff
@@ -56,7 +86,7 @@ get_branch_name <- function() {
 }
 
 get_remote_name <- function() {
-  git2r::branch_remote_name(git2r::branch_get_upstream(git2r::repository_head()))
+  git2r::branch_remote_name(git2r::branches(flags = "remote")[[1]])
```

Comment:
We should maintain the remote name for the active branch if it has been pushed already. Otherwise, use `origin` if exists, or fail.